README.md

```diff
@@ -1,229 +1,19 @@
-# High-Performance Material Point Method (CB-Geo mpm)
-> [CB-Geo Computational Geomechanics Research Group](https://www.cb-geo.com)
+# High-Performance Material Point Method
```

Comment:
I am rooting for a new name 😃 What do you think?

Reply:
"Super High-Performance Material Point Method"

Reply:
:rocket:

Reply:
To make it like a fraternity "MΠM"